### ISSUE TITLE
Re-Planning

### ISSUE THREAD

**Issue by GuyBenhaim** [google/or-tools#850]
After Solver creates routes for all the vehicles to serve certain M orders, now N new orders are added and Solver should run again to include these in an updated solution. Expectations:

**a.** All vehicles/routes should serve the same original M Orders. None of these M Orders can be dropped.
**b.** The N new orders should be added to the most fitting vehicles/routes.
**c.** If the new N orders are high-priority can we have an option to drop some/all of the M orders?
**d.** In case we originally used TW, can we force Solver to keep the time windows for the original M orders?  (new N will be only in slack times)
**e.** If originally we had no TW constraint (so no slack), can Solver keep only the assignments of route to the M orders, and only change visit times to fit-in the N  new Orders?

Basically, how can we use the original solution as a basis for re-running the Solver?
As opposed to reflecting the original solution as multiple constraints and planning from 0.

**Comment by Mizux** [comment]
Did you try to give an initial route ?
https://developers.google.com/optimization/routing/routing_tasks#setting-initial-routes-for-a-search

**Comment by 270ajay** [comment]
1. Run solver with M orders and store them in array (https://developers.google.com/optimization/routing/routing_tasks#storing-solutions-in-an-array)

1. Use `AddSoftSameVehicleConstraint` with the M orders solution and reoptimize with 'M&N' orders. You can give initial routes if the 'M&N' solution is not good (@Mizux 's comment above). Initial routes should be feasible 

1. Give cost for each order and make them optional using `AddDisjunction` (@Mizux 's comment above)

1. Use the usual time window constraint and keep loose time window for N orders (https://developers.google.com/optimization/routing/cvrptw#time_window_constraints)

1. `AddSoftSameVehicleConstraint` would do that
